src/main/java/org/apache/accumulo/testing/performance/tests/HighSplitCreationPT.java

```diff
@@ -36,6 +35,7 @@
   private static final int ONE_SECOND = 1000;
   private static final String TABLE_NAME = "highSplitCreation";
   private static final String METADATA_TABLE_SPLITS = "123456789abcde";
+  private static final String METADATA_TABLE_NAME = "accumulo.metadata";
```

Comment:
I think this is fine but we could probably put the names in a shared place of constants and reuse them.

Reply:
> I think this is fine but we could probably put the names in a shared place of constants and reuse them.

Good idea. Do you know of what a good place for these might be? @milleruntime

Reply:
TestProps.java seems like a good location.

Reply:
Addressed in c8b6cf4